src/components/header/header.component.js

```diff
@@ -1,12 +1,54 @@
 import React, { Component } from 'react'
+import cn from 'classnames'
+
 import logo from '../../images/logo.png'
 import './header.css'
+import MenuToggle from '../menu/toggle.component.js'
+import Menu from '../menu/menu.component'
 
 class Header extends Component {
+  constructor(props) {
+    super(props)
+
+    this.state = {
+      menuOpen: false
+    }
+
+    this.handleClose = this.handleClose.bind(this)
+    this.handleOpen = this.handleOpen.bind(this)
+  }
```

Comment:
we can use class properties instead of constructors

```
class Header extends Component {
  state = {
    menuOpen = false
  }

  handleClose = () => { ... }
  handleOpen = () => { ... }

  ...
}
```

Reply:
👍